### ISSUE TITLE
Question: using darktable-chart without physical chart card

### ISSUE THREAD

**Issue by stakaz** [darktable-org/darktable#6684]
Hello, I wonder if I could somehow get darktable-chart to work without having a colorchecker card or another physical thing I can take a photo of. 

My goal is to get close-to-camera-jpg result from a RAW file as a starting point. I am usually quite happy with the results of my Canon M6 but when it comes to editing, it would be great to work with raw for wider ranges and so on. For this I would like to create a style with darktable-chart. But I do not have a color chart and cannot spend money on it.

There are some styles for this camera already but they are somehow not very satisfying.

Would it somehow be possible to use a general image pair from camera to create such a thing with darktable-chart? 

I could imagine to take a photo of a monitor with a color chart and use it somehow by defining which corner has to be the white and black and so on. I can imagine, that to generate o tone curve and a color-map from raw to jpg in principle only relative deviations between these two are needed, am I right? 

This approach will for sure **not** be optimal but would it be possible to get close? 

I am not sure whether a github issue is the best thing to ask such a question but I do not know for anything better.

Thanks.

**Comment by MStraeten** [comment]
darktable chart just creates a style consisting of a color lut and a tonecurve that can be used to process the source in darktable resulting in the target.
If your source is a raw file and the target is the ooc jpeg then the style mimics the processing in camera - but just for the conditions valid when creating the style. If you target is the specification of the chart the source must be a shot of this chart.
So you can use a shot of a card on your monitor just to mimic the in camera processing - but this will be not very satisfying. At least as long as your monitor can’t provide more dynamic range than your camera is able to capture. 
Dont expect such styles to be useful in combination with filmic. These styles depends on color lookup table and the calculated tonecurve to achieve the correct colors. But filmic is between them in the pixel pipe, so the tonecurve is applied to conditions it wasn’t calculated for ...

**Comment by stakaz** [comment]
Hello, thanks for the answer. I ave experimented a bit and was able to produce results which at least are good enough as a starting point. However, now after I have watched some tutorials etc. I wonder if there is another way to adjust the colors but still be able to use filmic. I kind of would like to do the same thing darktable-chart is doing but without the loose of RGB-workflow. Is there something coming for this purpose? Maybe darktable-chart  but with 3D LUT?

In other words, how one can use darktable-chart and filmic simultaneously?

**Comment by matt-maguire** [comment]
You could produce an ICC profile by taking a shot of a color target and using argyllCMS:
https://encrypted.pcode.nl/blog/index.html%3Fp=594.html

Otherwise, I just suggest you decide what is it you like about the JPEGs you are getting, and replicate it using exposure/filmic, color balance, local contrast, etc. and create a style for it.

**Comment by stakaz** [comment]
Thanks for the reply.

> You could produce an ICC profile by taking a shot of a color target and using argyllCMS:
> https://encrypted.pcode.nl/blog/index.html%3Fp=594.html

Thanks, so that would essentially be something like LUT but without the tone curve?

> Otherwise, I just suggest you decide what is it you like about the JPEGs you are getting, and replicate it using exposure/filmic, color balance, local contrast, etc. and create a style for it.

Yes, probably this is the best way to go, I just thought,that there is a "more straight forward" approach...

**Comment by aurelienpierre** [comment]
Please use forums for question like these.

Eyeballing a color correction is most likely to fail, human eyes are very poor at keeping a reference and will adapt to surround lighting and color
